core/tox.ini

```diff
@@ -4,7 +4,7 @@ envlist =
 
 [testing]
 deps =
-    grpcio
+    grpcio >= 1.0.2rc0
     mock
```

Comment:
Changing the version would have to be a deliberate thing anyway and the CI would have to pass before a release.

Pinning this now would keep us from worrying about the future `latest grpcio` being flakey. If grpcio does become flakey down the road we could pin this at that time.